.github/workflows/duplicate-app-studio-service-issue.yml

```diff
@@ -0,0 +1,28 @@
+name: duplicate issue check for app-studio-service 
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v2
+        with:
+          repository: "OfficeDev/teamsfx-issue-triage-bot"
+          ref: main
```

Comment:
we always make code change on dev, and looking forward to revisions and updates.
so this ref may be dev? so checkout the latest change in actions files?

Reply:
The OfficeDev/teamsfx-issue-triage-bot repo is a simple project.
It is better to use the stable code in main branch.